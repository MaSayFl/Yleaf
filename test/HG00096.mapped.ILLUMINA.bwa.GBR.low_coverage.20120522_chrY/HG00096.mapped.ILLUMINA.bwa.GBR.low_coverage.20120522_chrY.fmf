chr	marker_name	haplogroup	pos	mutation	anc	der	reads	called_perc	called_base	state	Description
chrY	M176	O1b2	2787139	G->A	G	A	1	100	T	NA	Discordant genotype
chrY	L1378	D2	2960099	C->T	C	T	2	100	A	NA	Discordant genotype
chrY	P201	O2a2	2960155	T->C	T	C	3	100	A	NA	Discordant genotype
chrY	M343	R1b	3019783	C->A	C	A	6	100	G	NA	Discordant genotype
chrY	V38	E1b1a	6950250	C->T	C	T	3	100	G	NA	Discordant genotype
chrY	V13	E1b1b1a1b1a	6974222	G->A	G	A	2	100	T	NA	Discordant genotype
chrY	V32	E1b1b1a1a1b	7064780	G->C	G	C	4	100	T	NA	Discordant genotype
chrY	JST-002611	O2a1c	7678685	G->A	G	A	3	100	C	NA	Discordant genotype
chrY	F6251	D1a	7813234	C->T	C	T	1	100	A	NA	Discordant genotype
chrY	P326	LT	8599249	T->C	T	C	2	100	A	NA	Discordant genotype
chrY	P256	M	8817190	G->A	G	A	2	100	T	NA	Discordant genotype
chrY	Z284	R1a1a1b1a3a	8849155	C->G	C	G	1	100	A	NA	Discordant genotype
chrY	Z282	R1a1a1b1a	13476521	T->C	T	C	7	100	G	NA	Discordant genotype
chrY	L21	R1b1a1a2a1a2c1	13542548	C->G	C	G	1	100	A	NA	Discordant genotype
chrY	M438	I2	14526924	A->G	A	G	9	100	C	NA	Discordant genotype
chrY	V168	A1	15835792	G->A	G	A	1	100	T	NA	Discordant genotype
chrY	Z1936	N1a1a1a1a2	19301440	C->T	C	T	1	100	A	NA	Discordant genotype
chrY	M13	A1b1b2b	19560212	G->C	G	C	2	100	A	NA	Discordant genotype
chrY	M123	E1b1b1b2a1	19602700	C->T	C	T	1	100	G	NA	Discordant genotype
chrY	M122	O2	19602788	A->G	A	G	2	100	C	NA	Discordant genotype
chrY	M96	E	19617112	C->G	C	G	1	100	A	NA	Discordant genotype
chrY	M89	F	19755427	C->T	C	T	1	100	G	NA	Discordant genotype
chrY	M285	G1	20579854	G->C	G	C	1	100	T	NA	Discordant genotype
chrY	M304	J	20587967	A->C	A	C	3	100	T	NA	Discordant genotype
chrY	M526	K2	21389038	A->C	A	C	2	100	T	NA	Discordant genotype
