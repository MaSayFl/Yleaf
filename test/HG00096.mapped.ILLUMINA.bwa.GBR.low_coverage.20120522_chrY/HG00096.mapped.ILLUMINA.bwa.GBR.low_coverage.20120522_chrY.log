Total of reads: 14219874
Valid markers: 64665
Markers with zero reads: 0
Markers below the read threshold {1}: 0
Markers below the base majority threshold {50}: 0
Markers with discordant genotype: 381
Markers without haplogroup information: 381
Markers with haplogroup information: 64284
