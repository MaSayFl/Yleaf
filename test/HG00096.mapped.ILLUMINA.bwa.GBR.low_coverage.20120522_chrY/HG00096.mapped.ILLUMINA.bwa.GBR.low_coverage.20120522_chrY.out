chr	pos	marker_name	haplogroup	mutation	anc	der	reads	called_perc	called_base	state
chrY	6920150	V148	A0	G->A	G	A	3	100	A	D
chrY	2958271	L1086	A00	A->T	A	T	2	100	T	D
chrY	19577868	M31	A1a	G->C	G	C	2	100	G	A
chrY	19578550	M32	A1b1b	T->C	T	C	2	100	T	A
chrY	19704954	M42	BT	A->T	A	T	3	100	T	D
chrY	2866813	M130	C	C->T	C	T	2	100	T	D
chrY	19580272	M38	C1b2a	T->G	T	G	1	100	T	A
chrY	3009438	M347	C1b2b	A->G	A	G	3	100	A	A
chrY	13325453	M217	C2	A->C	A	C	1	100	C	D
chrY	19587995	M48	C2b1a2	A->G	A	G	3	100	A	A
chrY	19710852	M55	D1b	T->C	T	C	2	100	C	D
chrY	19578564	M33	E1a	A->C	A	C	1	100	C	D
chrY	13355944	M215	E1b1b	A->G	A	G	3	100	G	D
chrY	19910211	P287	G2	G->T	G	T	4	100	G	A
chrY	16635613	M436	I2a2	G->C	G	C	3	100	G	A
chrY	20579932	M267	J1	T->G	T	G	1	100	T	A
chrY	19571568	M20	L	A->G	A	G	5	100	A	A
chrY	13357844	M231	N	G->A	G	A	1	100	A	D
chrY	19816895	B479	N1a1a1a1a1c~	C->A	C	A	3	100	A	D
chrY	3012505	B202	N1a1a1a1a3b	T->C	T	C	3	100	T	A
chrY	16968722	F2930	N1b	G->A	G	A	2	100	G	A
chrY	7700527	P186	O	C->A	C	A	2	100	A	D
chrY	19600799	M119	O1a	T->G	T	G	2	100	G	D
chrY	16984483	M3	Q1b1a1a	G->A	G	A	3	100	A	D
chrY	13470103	M207	R	A->G	A	G	1	100	G	D
chrY	19995425	P312	R1b1a1a2a1a2	C->A	C	A	2	100	A	D
chrY	4994820	V88	R1b1a2	C->T	C	T	3	100	T	D
